pkg/gpu/aggregator.go

```diff
@@ -38,13 +38,13 @@ type aggregator struct {
 	// processTerminated is true if the process has ended and this aggregator should be deleted
 	processTerminated bool
 
-	// sysCtx is the system context with global GPU-system data
-	sysCtx *systemContext
+	// deviceMaxThreads is the maximum number of threads the GPU can run in parallel, for utilization calculations
```

Comment:
is this parameter is the same for ALL gpu devices available on specific host?

Reply:
No, but each aggregator only deals with a single device. This way we remove that dependency on the `systemContext`.